src/Components/Facility/AddInventoryForm.tsx

```diff
@@ -61,6 +60,7 @@ export const AddInventoryForm = (props: any) => {
   const [data, setData] = useState<Array<InventoryItemsModel>>([]);
   const [currentUnit, setCurrentUnit] = useState<any>();
   const [facilityName, setFacilityName] = useState("");
+  const [optionLabelId, setOptionLabelId] = useState("");
```

Comment:
Why a seperate state? Why not use the reducer alone?

Reply:
if i am using the reducer it generate error